k8s/redis/deployment.yml

```
@@ -0,0 +1,20 @@
+      app: redis
+  template:
+    metadata:
+      labels:
+        app: redis
+    spec:
+      containers:
+      - name: redis
+        image: redis:alpine
+        ports:
+        - containerPort: 6379
+---
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)